VRDR.CLI/Program.cs

```diff
@@ -41,7 +41,8 @@ static int Main(string[] args)
                 deathRecord.CertificationRole = certificationRole;
 
                 // State Local Identifier
-                deathRecord.StateLocalIdentifier = "42";
+                Tuple<string, string>[] ids = new Tuple<string, string>[]{Tuple.Create(ExtensionURL.AuxiliaryStateIdentifier1, "000000000042")};

```

Comment:
THis looks like a syntactic step backward.
Should be deathRecord.StateLocalIdentifier1 = 
                  deathRecordStateLocalIdentifier2 = 
                  
           The two identifiers are not an array, but rather two separate extensions.